<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1084</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Database Tool</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>12</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QRadioButton" name="rdbtnMysql">
        <property name="text">
         <string>mysql</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rdbtnXml">
        <property name="text">
         <string>xml</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QToolBox" name="tbMain">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="pgIniFile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1066</width>
         <height>384</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Ini File</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ini File</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtIniFile">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnIniFileChoice">
            <property name="text">
             <string>choice</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="lblDbName">
            <property name="text">
             <string>database</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtDatabase"/>
          </item>
          <item>
           <widget class="QLabel" name="lblServerOrFile">
            <property name="text">
             <string>server</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtServerOrFile"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="lblUser">
            <property name="text">
             <string>user name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtUser"/>
          </item>
          <item>
           <widget class="QLabel" name="lblPassWord">
            <property name="text">
             <string>password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edtPassWord"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="pgXmlFile">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1066</width>
         <height>384</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Xml File</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>1061</width>
          <height>380</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,1,4,4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>xml File</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtXmlFile">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnXmlFileChoice">
             <property name="text">
              <string>choice</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>prompty value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtPromValue"/>
           </item>
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="cboPromType">
             <item>
              <property name="text">
               <string>identity</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>unique</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="edtInsertPrompty">
             <property name="text">
              <string>insert</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>table name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtTableName"/>
           </item>
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>prompty list</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="edtPromList"/>
           </item>
           <item>
            <widget class="QPushButton" name="btnInsertTable">
             <property name="text">
              <string>insert</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>database name</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="edtDbName"/>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>table list</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTextEdit" name="edtTableList"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="bbMain">
      <property name="standardButtons">
       <set>QDialogButtonBox::Apply</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
